applications/luci-app-mwan3/htdocs/luci-static/resources/view/mwan3/status/overview.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+'require poll';
+'require view';
+'require rpc';
+
+var callMwan3Status = rpc.declare({
+	object: 'mwan3',
+	method: 'status',
+	expect: {  },
+});
+
+document.querySelector('head').appendChild(E('script', {
+	'type': 'text/javascript',
+	'src': L.resource('view/mwan3/mwan3.js')
```

Comment:
You should consider moving the parts of `'view/mwan3/mwan3.js'` which you use in multiple views into a separate helper class, like [`tools/firewall.js`](https://github.com/openwrt/luci/blob/master/applications/luci-app-firewall/htdocs/luci-static/resources/tools/firewall.js) or [`tools/network.js`](https://github.com/openwrt/luci/blob/master/modules/luci-mod-network/htdocs/luci-static/resources/tools/network.js).

Once you did that, you can simply do `'require tools.mwan3 as mwan3';` and use its `mwan3`'s methods instead of manually injecting a script here.

Reply:
Thanks for pointing that out, I didn't know.
However, I can now manage without this require file.
But good to know for next time